src/prefect/cli/cloud.py

```diff
@@ -35,6 +47,74 @@
 app.add_typer(cloud_app)
 
 
+# Set up a little API server for browser based `prefect cloud login`
+
+
+def set_login_api_ready_event():
+    login_api.extra["ready-event"].set()
+
+
+login_api = FastAPI(on_startup=[set_login_api_ready_event])
+
+login_api.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+class LoginSuccess(BaseModel):
+    api_key: str
+    workspace_id: UUID
+    account_id: UUID
```

Comment:
These will be dropped from the payload; we have decided the UI should not include a workspace selector.